maven-embedder/src/main/java/org/apache/maven/cli/CLIManager.java

```diff
@@ -134,7 +134,7 @@ public CLIManager()
         options.addOption( Option.builder( FAIL_FAST ).longOpt( "fail-fast" ).desc( "Stop at first failure in reactorized builds" ).build() );
         options.addOption( Option.builder( FAIL_AT_END ).longOpt( "fail-at-end" ).desc( "Only fail the build afterwards; allow all non-impacted builds to continue" ).build() );
         options.addOption( Option.builder( FAIL_NEVER ).longOpt( "fail-never" ).desc( "NEVER fail the build, regardless of project result" ).build() );
-        options.addOption( Option.builder( RESUME_FROM ).longOpt( "resume-from" ).hasArg().desc( "Resume reactor from specified project" ).build() );
+        options.addOption( Option.builder( RESUME_FROM ).longOpt( "resume-from" ).hasArg().optionalArg( true ).desc( "Resume reactor from last failed project or specified project" ).build() );
```

Comment:
@rfscholte, should the message read "last failed **project**" or "last failed **module**"? I still find myself struggling with terminology every now and then...

Reply:
Module might cause more confusion due to JPMS. However it is a Maven MultiModule project... sorry, didn't answer your question ;)

Reply:
'Project' was already there, I've at least wanted to keep it consistent. Let me know if we want module instead, I'll change it gladly

Reply:
In that case I would vote to stay with "project".